contrib/inventory/vmware_inventory.py

```diff
@@ -494,16 +494,13 @@ def instances_to_inventory(self, instances):
             for k, v in inventory['_meta']['hostvars'].items():
                 if 'customvalue' in v:
                     for tv in v['customvalue']:
-                        if not isinstance(tv['value'], string_types):
```

Comment:
Why is this being removed?

Reply:
When groupby_custom_fields is used, customValues can be of type `vim.CustomFieldsManager.Value`. Due to this block of code, script does not go beyond this point. I am not sure why this was added.